src/Apps/Conversation/routes.tsx

```diff
@@ -0,0 +1,47 @@
+import { graphql } from "react-relay"
+
+// @ts-ignore
+import { RouteConfig } from "found"
+import { ConversationAppFragmentContainer as ConversationApp } from "./ConversationApp"
+import { DetailFragmentContainer as DetailRoute } from "./Routes/Detail"
+
+export const routes: RouteConfig[] = [
+  {
+    path: "/user/conversations",
+    Component: ConversationApp,
```

Comment:
Following patterns [here](https://github.com/artsy/reaction/blob/master/src/Apps/Artist/routes.tsx#L65) can be we sure to split this bundle? Note that an additional [`export default`](https://github.com/artsy/reaction/blob/master/src/Apps/Artist/ArtistApp.tsx#L147-L148) will need to be used inside of the component that's being loaded.